Cargo.toml

```
@@ -200,6 +200,10 @@ path = "examples/3d/pbr.rs"
 name = "shadow_biases"
 path = "examples/3d/shadow_biases.rs"
 
+[[example]]
+name = "3d_shapes"
+path = "examples/3d/shapes.rs"
```

```suggestion
path = "examples/3d/3d_shapes.rs"
```

The problem with this is that if a user sees in the file tree `shapes.rs` thinks that the example name is `shapes`.

Remember to rename the file and change the reference to the README if you accept this.